### PR TITLE
Use the releases collections.

### PR DESCRIPTION
**Pull request by Natim**
- [x] Update the decoder to match the JSONSchema (Change null-able into optional)

### CODE REVIEW

**Comment by leplatrem** [comment]
Don't hesitate to adjust the fixtures to reflect the changes you're making here

File: ui/View.elm [hunk 1]
```diff
@@ -293,30 +293,7 @@ treeToMozillaHgUrl : Maybe String -> Maybe String
...
-                        , ( "oak", "projects/oak" )
-                        , ( "try-comm-central", "try-comm-central" )
-                        ]
-            in
-                Maybe.map
-                    (\folder ->
-                        "https://hg.mozilla.org/" ++ folder ++ "/rev/"
-                    )
-                    (Dict.get tree mappingTable)
+            Just <| "https://hg.mozilla.org/" ++ tree ++ "/rev/"
```
Comment (leplatrem) [inline ui/View.elm]: This is `repository`

**Comment by Natim** [comment]
This really slow down the application. Because loading the releases collection can takes about 23 seconds and loading them in memory then take

**Comment by Natim** [comment]
LGTM thanks.

**Pull request merged**